dts/bindings/ethernet/qca,ar8031.yaml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: Qualcomm AR8031 PHY
+
+compatible: "qca,ar8031"
```

Comment:
the qualcomm prefix is qcom not qca

Reply:
Actually, ar8031 is from Qualcomm Atheros, Inc., so I think we should use qca according to https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/vendor-prefixes.txt#L532

Reply:
I see, thanks for the correction